src/styles/inputs.js

```diff
@@ -0,0 +1,17 @@
+import { css } from 'styled-components';
+import theme from 'styles/theme';
+
+const inputBase = css`
+  border: 1px solid ${theme.color.light_grey};
+  border-radius: 4px;
+  color: ${theme.color.dark_gray};
```

Comment:
`dark_gray` is not in the `theme` variable. It's under `palette` only so this isn't actually having an effect.